internal/storage/storage.go

```diff
@@ -30,6 +30,10 @@ var (
 	ErrConnectionClosed = errors.New("connection is closed")
 	ErrTxClosed         = errors.New("transaction closed")
 	ErrTxCommitRollback = errors.New("transaction cannot commit or rollback")
+
+	ErrInvalidLatitude         = errors.New("latitude must be between -190,180")
```

Comment:
Are these error messages correct? Why -190?

Reply:
oh! Yes you are right, the latitude muss be between 90 and -90.